prime-router/src/main/kotlin/Sender.kt

```diff
@@ -6,14 +6,15 @@ import gov.cdc.prime.router.tokens.JwkSet
 
 /**
  * A `Sender` represents the agent that is sending reports to
- * to the data hub (minus the credentials used by that agent, of course). It
+ * the data hub (minus the credentials used by that agent, of course). It
  * contains information about the specific topic and schema that the sender uses.
  */
 open class Sender(
     val name: String,
     val organizationName: String,
     val format: Format,
     val topic: String,
+    val customerStatus: String,
```

Comment:
Same comment - everywhere this is used, lets use an enum instead of a string.

Reply:
Upated